include/onnxruntime/core/providers/context.h

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <core/session/onnxruntime_cxx_api.h>
+
+struct Context {
```

Comment:
could you document the definition and usage of this "Context"?
And, Context is a too general name, consider to give a more concrete name.

Reply:
Done - how about "CustomOpContext"?